### PR TITLE
Fix compareplot input df column mismatch after we adapted arviz

### LINKED ISSUE #3401: pm.compareplot fails as it is looking for lower case column name
Should be easy to fix, but maybe it is better to fix it in arviz.

### PR DESCRIPTION
**Pull request by junpenglao**
Close #3401.

### CODE REVIEW

File: pymc3/plots/__init__.py [hunk 1]
```diff
@@ -53,6 +52,38 @@ def wrapped(*args, **kwargs):
...
+    elif 'LOO' in comp_df.columns:
+        comp_df = comp_df.rename(index=str,
+                                 columns={'LOO': 'loo',
+                                          'pLOO': 'p_loo',
+                                          'dLOO': 'd_loo',
+                                          'SE': 'se',
+                                          'dSE': 'dse',
+                                          'shape_warn': 'warning'})
+    args[0] = comp_df
+    args = tuple(args)
```
Comment (canyon289) [inline pymc3/plots/__init__.py]: Why does args need to be coerced into a tuple? The list can be unpacked in line 85 as well right?

File: pymc3/plots/__init__.py [hunk 2]
```diff
@@ -53,6 +52,38 @@ def wrapped(*args, **kwargs):
...
 pairplot = map_args(az.plot_pair)
 
+# addition arg mapping for compare plot
+swaps = [
+    ('varnames', 'var_names')
+]
+@functools.wraps(az.plot_compare)
+def compareplot(*args, **kwargs):
+    args = list(args)
+    comp_df = args[0].copy()
```
Comment (canyon289) [inline pymc3/plots/__init__.py]: This assumes that comp_df is passed as the 0th arg. Does this cover the case where comp_df is passed a kwarg?

**Comment by junpenglao** [comment]
@canyon289 any other comments? Otherwise will merge.

**Comment by canyon289** [comment]
Sorry give me three hours and I'll do final review. Apologies on delay

**Comment by canyon289** [comment]
This looks fine to me, especially since its temporary code that is going to get removed in a couple releases!

**Pull request merged**